contracts/vesting/src/vesting/account/mod.rs

```diff
@@ -32,6 +32,7 @@ pub struct Account {
     pub periods: Vec<VestingPeriod>,
     pub coin: Coin,
     storage_key: u32,
+    pub pledge_cap: Option<PledgeCap>,
```

Comment:
You need to put `#[serde(default)]` here so that it wouldn't blow up on trying to deserialize old accounts without defined pledge_cap